pkg/driverbuilder/builder/amazonlinux.go

```diff
@@ -212,7 +211,6 @@ func buildMirror(a amazonBuilder, r string, kv kernelrelease.KernelRelease) (str
 	}
 
 	mirror := fmt.Sprintf("%s/%s", baseURL, "mirror.list")
-	slog.With("url", mirror, "version", r).Debug("looking for repo...")
```

Comment:
I don't think we actually needed these ones; passing a `printer` down there would be a nightmare :) 
Also, i think distro builders should not log anything.